### ISSUE TITLE
Error running Veeam VBR report if AsBuiltReport.Core v.1.4.0 is installed

### ISSUE THREAD

**Issue by walkersway** [AsBuiltReport/AsBuiltReport.Veeam.VBR#150]
### Bug description

Right at the end i get the following message. 

VERBOSE: [ 17:52:26:894 ] [ Document ] - Total processing time '8.08' minutes.
New-AsBuiltReport : Cannot bind argument to parameter 'Style' because it is null.
At line:1 char:1
+ New-AsBuiltReport -Report Veeam.VBR -Target xxx.xxx.lo ...
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (:) [Write-Error], WriteErrorException
    + FullyQualifiedErrorId : Microsoft.PowerShell.Commands.WriteErrorException,New-AsBuiltReport

### Command-line input

New-AsBuiltReport -Report Veeam.VBR -Target xxxxx.xxxxx.local -Username 'xxxxx\xxxxx' -Password 'xxxxx' -Format Text,Html,Word -OutputFolderPath 'C:\Temp\asbuilt' -ReportConfigFilePath 'AsBuiltReport.Veeam.VBR.json' -Verbose

### Steps to reproduce

I have tried multiple json's for this. 

using Veeam 12, powershell as administrator. I run the command line and it runs though right until the end. I only get a Txt output, no word or HTML file

Example of one of the json files

{
    "Report": {
        "Name": "Veeam Backup & Replication As Built Report",
        "Version": "1.0",
        "Status": "Released",
        "ShowCoverPageImage": true,
        "ShowTableOfContents": true,
        "ShowHeaderFooter": true,
        "ShowTableCaptions": true
    },
    "Options": {
        "BackupServerPort": 9392,
        "EnableCharts": false,
        "EnableHardwareInventory": false,
        "EnableDiagrams": false,
        "PSDefaultAuthentication": "Default"
    },
    "InfoLevel": {
		"_comment_": "Please refer to the AsBuiltReport project contributing guide for information about how to define InfoLevels.",
        "_comment_": "0 = Disabled, 1 = Enabled, 2 = Adv Summary, 3 = Detailed",
        "Infrastructure": {
            "BackupServer": 3,
            "BR": 3,
            "Licenses": 3,
            "Proxy": 3,
            "Settings": 3,
            "SOBR": 3,
            "ServiceProvider": 3,
            "SureBackup": 3,
            "WANAccel": 3

        },
        "Tape": {
            "Library": 1,
            "MediaPool": 1,
            "NDMP": 1,
            "Server": 1,
            "Vault": 1
        },
        "Inventory": {
            "FileShare": 1,
            "PHY": 1,
            "VI": 1
        },
        "Storage": {
            "ISILON": 1,
            "ONTAP": 1
        },
        "Replication": {
            "FailoverPlan": 3,
            "Replica": 3
        },
        "CloudConnect": {
            "BackupStorage": 1,
            "Certificate": 1,
            "CloudGateway": 1,
            "GatewayPools": 1,
            "PublicIP": 1,
            "ReplicaResources": 1,
            "Tenants": 1
        },
        "Jobs": {
            "Agent": 1,
            "Backup": 3,
            "BackupCopy": 3,
            "FileShare": 1,
            "Surebackup": 1,
            "Replication": 3,
            "Restores": 3,
            "Tape": 1
        }
    },
    "HealthCheck": {
        "Infrastructure": {
            "BackupServer": true,
            "Proxy": true,
            "Settings": true,
            "BR": true,
            "SOBR": true,
            "Server": true,
            "Status": true,
            "BestPractice": true
        },
        "Tape": {
            "Status": false,
            "BestPractice": false
        },
        "Replication": {
            "Status": true,
            "BestPractice": true
        },
        "Security": {
            "BestPractice": false
        },
        "CloudConnect": {
            "Tenants": false,
            "BackupStorage": false,
            "ReplicaResources": false,
            "BestPractice": false
        },
        "Jobs": {
            "Status": true,
            "BestPractice": true
        }
    }
}

### Expected behaviour

Word and HTML file would be created. 

### Screenshots

I have copied the relevant error. I have never had any errors on any other veeam backup and replication server before. 

### Operating System

Windows 2019

### PowerShell Version

PS H:\> $psversiontable

Name                           Value
----                           -----
PSVersion                      5.1.17763.5328
PSEdition                      Desktop
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0...}
BuildVersion                   10.0.17763.5328
CLRVersion                     4.0.30319.42000
WSManStackVersion              3.0
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1

### PowerShell Modules

Name                    Version
----                    -------
AsBuiltReport.Core      1.4.0
AsBuiltReport.Veeam.VBR 0.8.5
PScribo                 0.10.0

### Before submitting

- [X] I have read and followed the [bug reporting guidelines](https://www.asbuiltreport.com/about/contributing/#reporting-issues-and-bugs).
- [X] I have read [the documentation](https://www.asbuiltreport.com/user-guide/new-asbuiltconfig), and referred to the [known issues](https://www.asbuiltreport.com/user-guide/known-issues/) before submitting this bug report.
- [X] I have checked for previously opened & closed [issues](https://github.com/AsBuiltReport/AsBuiltReport.Veeam.VBR/issues) before submitting this bug report.

**Comment by walkersway** [comment]
Veeam build number is 12.0.0.1420

**Comment by rebelinux** [comment]
Hello,

Remove AsBuiltReport.Core v1.4.0 and install v1.3.0

```powershell
get-module -Name AsBuiltReport.Core -ListAvailable | uninstall-Module -Force
install-module AsBuiltReport.Core -MaximumVersion 1.3.0
import-module AsBuiltReport.Core -Force
```
after that try to run the report again.

Hope this helps!

**Comment by walkersway** [comment]
Hi


I still have access to the server I was running it on. I have a meeting tomorrow but will give it a try when I get back and let you know the outcome.

Thank you.

Sent from Nine<http://www.9folders.com/>
________________________________

**Comment by walkersway** [comment]
Hi

Thank you so much i can confirm this worked and HTML and Word versions were created. Thank you.

**Comment by rebelinux** [comment]
This bug will be fixed in v0.8.6.

[Veeam Backup & Replication As Built Report.pdf](https://github.com/AsBuiltReport/AsBuiltReport.Veeam.VBR/files/15119829/Veeam.Backup.Replication.As.Built.Report.pdf)
